docs/ecu-ElmProtocolTelegramProtocol.js

```
@@ -0,0 +1,286 @@
+
+        if (this.lastCommand) {
+          const cmdToRetry = this.lastCommand;
+
+          Promise.resolve().then(() => this.cmdQueue.push(cmdToRetry));
+        }
+
+        return false;
+      }
+
+      case RSP_ID.MODEL:
+        const initSuccess = await this.initialize();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)